### ISSUE TITLE
Exclude Videos Only on Select Sites

### ISSUE THREAD

**Issue by BattouSaeen** [extesy/hoverzoom#1464]
**Is your feature request related to a problem? Please describe.**
On some sites, it's useful to have hoverzoom active for images, but not for the video. For example, many sites have an HTML video (like a trailer) that you can full screen but keeping the mouse in the area will trigger hoverzoom which causes 2 instances of the same video to play at the same time.

**Describe the solution you'd like**
A simple function/option or separate exclusion list for videos only. I'd like to keep hoverzoom active for both pictures and videos on some sites, while disabling only videos and keeping pictures still zoomable on other sites.

**Describe alternatives you've considered**
I've tried excluding the original URL of where the videos are uploaded and this kinda sorta works. It always zooms initially until mouse is moved a bit, then it doesn't trigger (it feels like a processing delay ... I already have a short delay for hoverzoom in general so this may exacerbate it). However, on some other sites this doesn't work at all even though URL is there, which makes me think those sites have some kind of obfuscation that stops hoverzoom from identifying that original URL.

**Comment by PowerUser64** [comment]
I agree, something like this would be greatly appreciated. Hoverzoom often gets in the way when I play videos that are already displaying through my browser's builtin video player. When I click the full screen button button in the embeded video player, hoverzoom pops up because I'm hovering on something that it wants to show, and it ends up playing twice.

For example, try watching the video here: https://github.com/echasnovski/mini.operators#demo
(I couldn't remember where any good examples with audio are, but it's worse when the audio plays from hoverzoom while it plays from the video.)

I think a whitelist for sites I want it to work on would be very helpful, since when it comes to videos, I find hoverzoom to be most helpful with sites like reddit that have a video on them that hoverzoom can extract without me needing to click through to the site.

I should also note, my browser is Firefox.

**Comment by Creomortis** [comment]
I'm experiencing this issue daily on reddit, double click a video to full screen and the audio plays twice out of sync. Very frustrating.

Example url:
https://www.reddit.com/r/UFOs/comments/1ipw162/ufo_seen_tonight_on_oklahoma_news/
